app/aws_meter.go

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
```

Comment:
Is there any way to add tests to this service without a ton of effort mocking things? maybe for the `getUserCategoryUsage`?